app/helpers/federation_helper.rb

```diff
@@ -234,7 +238,7 @@ def counts_ontology_ids_by_portal_name(portals_ids)
       counts[current_portal.downcase] += 1 if id.include?(current_portal.to_s.downcase)
 
       federation_portals.each do |portal|
-        portal_api = federated_portals[portal.downcase.to_sym][:api]
+        portal_api = federated_portals[portal.downcase.to_sym][:name].downcase
```

Comment:
Using a portal's name does not always work. Sometimes, the API URL is not the same as the portal name. 
And what did you need to change this line?

Reply:
In the search page, to count the results that are in the `:other_portals` list, we don't have the api url, but instead we have the ui url. 
Ok no big deal, to solve this I will just check both like this:
`counts[portal.downcase] += 1 if (id.include?(portal_api) || id.include?(portal_ui))`